tests/generate_fixtures/generate_target_dnadiff_files.py

```diff
@@ -71,11 +81,14 @@
     file.unlink()
 for file in SHOW_COORDS_DIR.glob("*.mcoords"):
     file.unlink()
+for file in DNADIFF_DIR.glob("*.report"):
+    file.unlink()
```

Comment:
Using a temp directory via ``tempfile`` would take care of all this cleanup automatically.

Reply:
Is this "cleanup" before running the tool mean to remove any pre-existing fixtures? This isn't needed if we're going to replace files, but makes sense if we were changing the example set (e.g. alternative genomes).

Reply:
I was about to ask a question on this. This line removes any pre-existing fixtures, so each time the script is re-run, it deletes the existing fixtures and generates new ones, regardless of whether the example set of genomes has changed or not. This part of code was introduced by someone (I assume either you or @widdowquinn) for other fixtures like `.mcoords` and `.rdiff` files. I simply expanded this logic for `.report`. I'm happy to make changes if needed, but I am a little unsure of what the best practise would be.

Removing this part of the code and re-running it on the same set of genomes doesn't break anything, as it simply overwrites the existing files.

Perhaps keeping this would be a good idea because re-running the scripts, even when the set of input genomes doesn't change, does no harm. However, it might complicate things when we decide to change the set of genomes. What do you think?

Reply:
Here is the commit that introduced the clean up of fixtures when regenerating them 57cdd6fd472c4c9afcae1656893acb09f8a2c32f. Are you still happy to keep it that way :)

Reply:
It was me then. 😩

Yes, I think it makes sense particularly for the use case of regenerating the fixtures as part of changing the input genomes. But we should perhaps add a comment about this (and ought to do this to the other scripts too)?

Reply:
It makes sense! I have added a comment now.

Regarding the changes to the other script, would you like me to make those changes in this branch or perhaps in a new one?

Reply:
A small separate PR would be safest - and any conflicts wouldn't hold up the dnadiff work